pipenv/core.py

```diff
@@ -2033,13 +2033,16 @@ def do_shell(three=None, python=False, fancy=False, shell_args=None, pypi_mirror
     click.echo("Launching subshell in virtual environment…", err=True)
 
     fork_args = (project.virtualenv_location, project.project_directory, shell_args)
-
+    dotenv_file = environments.PIPENV_DOTENV_LOCATION or os.sep.join(
+        [project.project_directory, ".env"]
+    )
```

Comment:
better to `os.path.join(project.project_directory, ".env")` instead

Reply:
Thanks! I normally would have used `os.path` but saw the sep thing elsewhere in the code and was trying to fit in :)